src/terminal/clear.lua

```diff
@@ -0,0 +1,114 @@
+--- Clear Module
```

Comment:
```suggestion
--- Clear Module.
```
adding a dot `.`, in the docs engine it separated the title from the body text.